Classes/Booking.cs

```diff
@@ -1,24 +1,29 @@
 ﻿namespace AirportTicketBooking.Classes
 {
-    internal class Booking
+    public class Booking
     {
         internal int BookingId { get; set; }
```

Comment:
why did you decide to use internal access modifier for the members?

Reply:
For this, one of my questions for the next meeting is how to decide the suitable access modifier. now I will change them into public